src/accelerator_structure.rs

```diff
@@ -346,3 +346,51 @@ impl IntersectionFunctionTableRef {
         unsafe { msg_send![self, setFunction: function atIndex: index] }
     }
 }
+
+// See https://developer.apple.com/documentation/metal/mtlvisiblefunctiontabledescriptor/
```

Comment:
Rustdoc and formatting:
```suggestion
/// See <https://developer.apple.com/documentation/metal/mtlvisiblefunctiontabledescriptor>
```